accumulo-store/src/main/java/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -250,7 +251,7 @@ protected void addAdditionalOperationHandlers() {
      * @throws StoreException failure to insert the elements into a table
      */
     public void addElements(final Iterable<Element> elements) throws StoreException {
-        TableUtils.ensureTableExists(this);
+        TableUtils.ensureTableExists(this); // Needed for tests that access store directly
```

Comment:
I think we should remove this check and force all tests to call initialise.
Better than wasting a check every addElementsCall.
Similarly you can remove the check from the BulkImport Operation